virtcontainers/kata_agent.go

```diff
@@ -1053,6 +1059,13 @@ func (k *kataAgent) handleBlockVolumes(c *Container) []*grpc.Storage {
 			continue
 		}
 
+		// Ensure the block device is formatted, for the devices here are specified as volumes
+		fsType, err := utils.GetDevFormat(m.Source)
+		if err != nil || fsType == "" {
+			k.Logger().WithField("device", id).WithError(err).Error("get device format failed")
+			return nil
+		}
```

Comment:
I'm curious you had got the right "fsType" here, why still pass the "bind" fsType here and  let
the agent re-parse the block's fsType again? Why not pass the right "fsType" here and the agent 
will do the right mount.

Reply:
Very good advice. I thought that agent does not perceive the "MountBlkInVM" config, we use this method to distinguish whether the blk device need mount in the guest VM to a directory. Now I'm using a more appropriate method, see my latest PR for details.